source/process/casestudy.rst

```diff
@@ -14,8 +14,12 @@ The process is approximately as follows:
 1. Customer has completed a deployment of Mattermost and is open to a case study. 
 2. Customer case study contact is introduced to a Mattermost case study coordinator over email and completes a questionnaire (below). 
 3. Mattermost case study coordinator arranges a 30-minute phone or video call with the customer for a discussion around the questionnaire and deployment. 
-4. One to two weeks later a draft case study is sent to the customer for review and approval. 
-5. After customer approval, the case study is published to the Mattermost blog and may be used in other marketing materials to share with other potential customers the benefits of a Mattermost solution.
+4. One to two weeks later a draft case study is sent to the customer for review and approval.
+5. Mattermost case study coordinator politely checks in with the customer every 2 business days for an estimate of the draft case study review time.
+6. The case study coordinator also commissions a banner with a clear heading and a nice background image to be included in the case study blog post.
```

Comment:
I agree the sample is good, but we might still want to clarify the meanings of "clear" and "nice"

Example: Should the image be a screenshot/photo of the company headquarters?